habitat/core/environments.py

```diff
@@ -73,25 +70,23 @@ def get_done(self, observations):
         done = False
         if self._env.episode_over:
             done = True
-        if self._rl_config.END_ON_SUCCESS and self._episode_success():
+        if self.config.TASK.END_ON_SUCCESS and self._episode_success():
             done = True
         return done
 
     def get_info(self, observations):
         return self.habitat_env.get_metrics()
 
 
-@baseline_registry.register_env(name="NavRLEnv")
+@habitat.registry.register_env(name="NavRLEnv")
 class NavRLEnv(habitat.RLEnv):
```

Comment:
This impacts all tasks, including non-rearrangement tasks (like PointNav/ObjectNav) are we okay with this?

Reply:
How big is the impact do you think? How may users make a distinction between the baseline registry and the habitat.registry?
I am having a hard time figuring it out.

Reply:
This should be fine. It'll impact users that are getting envs out of the baselines registry/registering their own envs, but it's a simple enough fix and this change overall makes sense.